src/components/selections/select.js

```diff
@@ -1,27 +1,40 @@
 'use strict';
 
 var polybool = require('polybooljs');
+var pointInPolygon = require('point-in-polygon/nested');
```

Comment:
FYI we already have an "is this point inside this polygon" in [`polygon.tester`](https://github.com/plotly/plotly.js/blob/ee8523138145451ae6d5800cbc400e02f0c1d87f/src/lib/polygon.js#L159)

Reply:
Good call. I tried to reuse it here: https://github.com/plotly/plotly.js/commit/f28576e44275dc6a4b2aea683a1a6b456b893dc7.
But it seems it require extra work.

Reply:
yeah it's old and undocumented... maybe I'll give it a shot myself at some point. Or maybe we drop that code entirely and just use `point-in-polygon` 😏